src/utils/routeHelpers.ts

```diff
@@ -161,10 +161,15 @@ const isAdmin = async (loggedUserID: number, neighborhoodID: number): Promise<bo
   return (neighborhood.admin_id === loggedUserID);
 };
 
+const isLoggedInAdmin = async (req: CustomRequest) => (
+  (req.user !== undefined) && await isAdmin(req.user!.id, Number(req.params.id))
+);
```

Comment:
Good thinking with function! Makes the code a lot cleaner.

Reply:
Nice bit of code, Antonina. Well conceived and well executed.

Reply:
thanks guys!!